cdq-analysis/main.go

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright 2023 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"go.uber.org/zap/zapcore"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/redhat-appstudio/application-service/cdq-analysis/pkg"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+func main() {
+	// remove the prefix and suffix quotes
+	for i := 1; i <= 10; i++ {
+		if strings.HasPrefix(os.Args[i], "\"") && strings.HasSuffix(os.Args[i], "\"") && len(os.Args[i]) > 1 {
+			os.Args[i] = os.Args[i][1 : len(os.Args[i])-1]
+		}
+	}
+	gitToken := os.Args[1]
```

Comment:
Do we need validation of the command-line args?

Reply:
the main.go is actually not being used by HAS controller yet. similarly, will be part of the docker image issue. may I add validation as part of that issue?

Reply:
Sure